introduction.md

```diff
@@ -1,3 +1,11 @@
 I am Glenn. It is an honor to be part of this dojo.
 I favor the JavaScript path of Code School.
 </bows>
+
+* HTML
+* CSS
+* JavaScript
+* jQuery
+* PHP
+
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?